src/utils/dialogmoterUtil.ts

```diff
@@ -0,0 +1,91 @@
+import {
+  DialogmoteDeltakerVarselType,
+  DialogmoterDTO,
+  DialogmoteStatus,
+} from "../data/dialogmoter/dialogmoterTypes";
+import { MoteRespons } from "../components/MoteResponsFilter";
+import { MoteDTO } from "../data/moter/moterTypes";
+
+export const isDialogmote = (
+  mote: MoteDTO | DialogmoterDTO
+): mote is DialogmoterDTO => (mote as DialogmoterDTO).sted !== undefined;
```

Comment:
Hmz, hvorfor sjekk på .sted? Vil ikke man her kunne få true på typeguard sjekk, men false på .sted? I så fall resolver den vel til false?

Reply:
Fant ingen bedre måte å sjekke om mote er av type DialogmoterDTO eller MoteDTO. sted-feltet er jo påkrevd for dialogmøter, men enig i at det ikke er helt vanntett. Kan grave litt i om det finnes lurere måte å gjøre type-sjekken på...